src/types/index.ts

```diff
@@ -13,7 +13,6 @@ export type PageInfo = {
 
 export type SavedItemConnection = {
   edges: SavedItemEdge[];
-  nodes: SavedItem[];
```

Comment:
@kschelonka shouldn't this stay?

Reply:
no the savedItem is inside edges.node, so we don't need it here

Reply:
yeah this was a mistake in the schema